fix(ChatbotFootnote): updated verbiage and combined examples (#846)
patternfly/chatbot: packages/module/patternfly-docs/content/extensions/chatbot/examples/UI/UI.md

## Footer

### Footnote

A `<ChatbotFootnote>` can be placed in the ChatBot footer to communicate any legal disclaimers or information about the ChatBot.

> The footnote can be static text by passing the `label` property, or it can be a button that triggers a popover by also passing the `popover` property.

Reviewer: ```suggestion
To create a static text footnote, pass the `label` property. To create a footnote button that triggers a popover, pass in both the `label` and `popover` properties.
```
assuming the latter requires both?
